minicluster/src/main/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -249,6 +250,17 @@ private void startMaster(TachyonConf testConf) throws IOException {
     // Update the test conf with actual RPC port.
     testConf.set(Constants.MASTER_PORT, String.valueOf(getMasterPort()));
 
+    // If we are using the LocalMiniDFSCluster, we need to update the UNDERFS_ADDRESS to point to
+    // the cluster's current address. This must happen here because the cluster isn't initialized
+    // until mMaster is started.
+    UnderFileSystemCluster ufs = UnderFileSystemCluster.get();
+    // TODO(andrew): Move LocalMiniDFSCluster to the integration-tests project so that we can use
+    // instanceof here instead of comparing classnames.
```

Comment:
`LocalMiniDFSCluster` isn't accessible from here, we need to move `LocalTachyonCluster` to the `integration-tests` project to use `instanceof`. The comment is referring to the wrong class - I'll update it.